scripts/typescript_checks.py

```diff
@@ -27,8 +27,761 @@
 
 from typing import Optional, Sequence
 
+import yaml
+
 from . import common
 
+# Contains the name of all files that are not strictly typed.
+# This list must be kept up-to-date; the changes (only remove) should be done
+# manually.
+# Please keep the list in alphabetical order.
+# NOTE TO DEVELOPERS: do not add any new files to this list without asking
+# @vojtechjelinek first.
+# pylint: disable=line-too-long, single-line-pragma
+NOT_FULLY_TYPE_STRICT_TSCONFIG_FILEPATH = [
```

Comment:
This variable name is confusing to me. Maybe `TS_STRICT_EXCLUDE_PATHS`?

Reply:
Done